app/views/shared/_navbar_new_button.html.erb

```diff
@@ -7,7 +7,7 @@
         flex items-center gap-2 rounded-md bg-white px-4 py-1 mt-1 text-sm font-semibold text-gray-900
         shadow-sm border border-gray-200 hover:bg-gray-50 active:bg-gray-100
         focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2
-        transition"
+        transition cursor-pointer"
       data-action="dropdown#toggle"
       data-dropdown-payload-param='[{"workshop-dropdown":"hidden"}]'
```

Comment:
i think we should rename the payload to `new-dropdown` or `new-button-dropdown`. doesn't have to be this PR.